strategy/to_json/from_serializable_test.ts

```diff
@@ -43,6 +43,33 @@ test({
   },
 });
 
+test({
+  name:
+    "fromSerializable - arrays of nested objects with an object set to null",
+  fn() {
+    class Test1 extends Serializable {
+      @SerializeProperty("nested_property")
+      serializeMe = 999;
```

Comment:
NIT: if we happen to go back for code changes in the PR could we add `public ` to these members, only for consistency, again it's not a blocker.

Reply:
I have it open still, 1 second...